src/include/nodes/execnodes.h

```diff
@@ -1522,6 +1540,9 @@ typedef struct ScanState
 	/* The type of the table that is being scanned */
 	TableType tableType;
 
+	/* Runtime filter */
+	struct RuntimeFilterState runtimeFilter;
```

Comment:
Since ScanState need to be copied some times, it's better to use a point of RuntimeFilterState in the struct and allocate memory dynamically.